src/components/plot/add/PlotAddSelectFinalDirectory.tsx

```diff
@@ -56,7 +56,7 @@ export default function PlotAddSelectFinalDirectory() {
           size="large"
           variant="outlined"
           selected={hasFinalLocation}
-          nowrap
+          nowrap={1}
```

Comment:
nowrap is boolean property

Reply:
1 was added so that react would not output a warning saying it isn't a boolean.
1= true

Reply:
nowrap alone is boolean by default

Reply:
> nowrap alone is boolean by default

react itself as a bug right now where it will give a warning if you don't do that

Reply:
@seeden do you still want me to change it?

Reply:
I might be able to get it now

Reply:
Got it!